File: ALGORITMO E PROGRAMAÇÃO/LISTA03/Exe01.js
const { number } = require("./functions_uteis")

function principal(){
    console.log(' ### Lista de números ###')
    // Entrada
    const numero = number('Informe um número: ')
    // Processamento
    for (let i = 1; i <= numero; i++){
        // Saída
        console.log(i)
    }
}
principal()

File: ESTRUTURA DE DADOS/LISTA02/Exe01.c
/*Leia uma lista de números e que para cada número lido, escreva o próprio número e a relação
de seus divisores. (flag número = 0).*/
#include <stdio.h>

int main(){
    int numero = 1;
    int divisores = 1;
    while(numero != 0){
      if(numero != 0){
        printf("Informe um número: ");
        scanf("%i",&numero);
        while(divisores <= numero){
          if(numero % divisores == 0){
            printf("\nOs divisores são: %i", divisores);
            printf("\n------------------------\n");
        }
        divisores++;
      };
      divisores = 1;
    };
    }
}
